src/lib/os-release.ts

```diff
@@ -0,0 +1,47 @@
+import * as Promise from 'bluebird';
+import * as fs from 'fs';
+import * as _ from 'lodash';
+
+// FIXME: Don't use synchronous file reading and change call sites to support a promise
+export function getOSReleaseField(path: string, field: string): string | undefined {
```

Comment:
Can you please search the code for references to this function and skip exporting it if it's not used? Or if it's easy, change call sites to use the higher-level exported functions below instead?